### PR TITLE
Document error classes

### PR DESCRIPTION
**Pull request by scottinet**
# Description

The error classes are missing from our framework documentation.

### CODE REVIEW

File: doc/2/framework/error-classes/partial-error/properties/index.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+# Properties
+
+| Name              | Type          | Description |
+|-------------------|---------------|-------------|
+| `code`         | <pre>number</pre> | Error unique code |
+| `id`           | <pre>string</pre> | Error unique name |
+| `name`         | <pre>string</pre> | Error class name (`PartialError`) |
+| `message`      | <pre>string</pre> | Error message  |
+| `stack`        | <pre>string</pre> | Error stack trace |
+| `status`       | <pre>number</pre> | Error status code (`206`) |
```
Comment (Aschen) [inline doc/2/framework/error-classes/partial-error/properties/index.md:17]:
```suggestion
| `status`       | <pre>number</pre> | Error status code (`206`) |
| `errors`       | <pre>Array&lt;KuzzleError&gt;</pre> | List of errors encountered when executing the action |
```

File: doc/2/framework/error-classes/plugin-implementation-error/constructor/index.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+type: page
+title: Constructor
+description: PluginImplementationError class constructor
+---
+
+# PluginImplementationError
+
+Inherits from [KuzzleError](/core/2/framework/abstract-classes/kuzzle-error/constructor).
+
+This error object is used on unexpected plugin errors.
```
Comment (Aschen) [inline doc/2/framework/error-classes/plugin-implementation-error/constructor/index.md:12]:
```suggestion
This error object is used on unexpected plugin or application errors.
```

**Review by Aschen — APPROVED** [review]
Do we really need to document the `toJson` method for every class implementing `KuzzleError` ?

**Review by Yoann-Abbes — APPROVED** [review]

**Comment by scottinet** [comment]
@Aschen > you're right, it makes no sense to redocument a method inherited from a parent class, if it isn't overloaded :+1:

**Pull request merged**